### PR TITLE
Refs #29722 -- Added introspection of materialized views for Oracle.

### PR DESCRIPTION
**Pull request by felixxm**
Ticket [29722](https://code.djangoproject.com/ticket/29722).

### CODE REVIEW

File: tests/inspectdb/tests.py [hunk 1]
```diff
@@ -310,16 +310,19 @@ def test_include_views(self):
...
     def test_include_materialized_views(self):
-        """inspectdb --include-views creates models for database materialized views."""
+        """
+        inspectdb --include-views creates models for database materialized
+        views.
+        """
         with connection.cursor() as cursor:
             cursor.execute(
-                'CREATE MATERIALIZED VIEW inspectdb_people_materialized_view AS '
+                'CREATE MATERIALIZED VIEW inspectdb_people_materialized AS '
```
Comment (felixxm) [inline tests/inspectdb/tests.py]: I changed the name of materialized view to fit in 30 characters limit in Oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

File: tests/inspectdb/tests.py [hunk 2]
```diff
@@ -310,16 +310,19 @@ def test_include_views(self):
             with connection.cursor() as cursor:
                 cursor.execute('DROP VIEW inspectdb_people_view')
 
-    @skipUnless(connection.vendor == 'postgresql', 'PostgreSQL specific SQL')
+    @skipUnlessDBFeature('supports_materialized_views')
     def test_include_materialized_views(self):
-        """inspectdb --include-views creates models for database materialized views."""
+        """
+        inspectdb --include-views creates models for database materialized
```
Comment (timgraham) [inline tests/inspectdb/tests.py]: If the line length bothers you, I think dropping "database" would be fine.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
@timgraham Thanks for the review :+1:

**Pull request merged**